packages/contracts/contracts/L1/messaging/L1StandardBridge.sol

```diff
@@ -28,7 +28,7 @@ contract L1StandardBridge is IL1StandardBridge, CrossDomainEnabled {
      * External Contract References *
      ********************************/
 
-    address public l2TokenBridge;
+    address public immutable l2TokenBridge;
```

Comment:
```suggestion
    address public immutable l2TokenBridge;
    address private __emptySlot; 
```
I think we need to do something like this to maintain the storage layout in the chugsplash proxy. 

Also, how do we go about testing the upgrade process to ensure this is done safely?

Reply:
Oh, looks like we might need to add another empty slot for the `messenger` variable which was made `immutable` in `CrossChainEnabled.sol`